src/components/TestDetailsDialog/TestDetailsModal.tsx

```diff
@@ -58,7 +58,7 @@ const defaultStagePos = {
   y: 0,
 };
 
-const useStyles = makeStyles((theme) => ({
+const useStyles = makeStyles(() => ({
```

Comment:
this has to fix Header part
![image](https://github.com/Visual-Regression-Tracker/frontend/assets/5182956/1ce4a214-80ea-4a96-b52c-7c213f9db463)

Reply:
Could you commit this change? Hard to copy values from image

Reply:
sure